src/batch-submitter/state-batch-submitter.ts

```diff
@@ -142,7 +146,12 @@ export class StateBatchSubmitter extends BatchSubmitter {
       'appendStateBatch',
       [batch, startBlock]
     )
-    if (!this._shouldSubmitBatch(tx.length / 2)) {
+    const batchSizeInBytes = tx.length / 2
```

Comment:
This needs to account for the `0x` prefix

Reply:
`ethers` has some utilities for hex data